src/Compilers/CSharp/Portable/Binder/BindingDiagnosticBag.cs

```diff
@@ -4,6 +4,8 @@
 
 #nullable enable
 
+global using BindingDiagnosticBag = Microsoft.CodeAnalysis.BindingDiagnosticBag<Microsoft.CodeAnalysis.CSharp.Symbols.AssemblySymbol>;
```

Comment:
To avoid not having to update all places in the codebase that refer to BindingDiagnosticBag (nakedly, to mean the C# subtype), i made this global-alias.  (note: need to put in an actual file for this purpose).

Withotu this, the impact would be much larger as every signature using BindingDiagnosticBag would need to update.  Given the desire to 'lower churn' this seemed expedient.